src/lib/services/simulator.ts

```diff
@@ -80,12 +86,20 @@ export async function checkRequirements(): Promise<Record<string, boolean>> {
   try {
     await checkCommand("docker --version", "docker");
     requirementsInstalled.docker = true;
-  } catch (error) {
+  } catch (error: any) {
     if (!(error instanceof MissingRequirementError)) {
       throw error;
     }
   }
 
+  if (requirementsInstalled.docker) {
+    try {
+      await checkCommand("docker ps", "docker");
+    } catch (error: any) {
+      await executeCommand(DEFAULT_RUN_DOCKER_COMMAND);
```

Comment:
I added this message ([here](https://github.com/yeagerai/genlayer-cli/pull/32/files#diff-cb56c9a3ea0c1a239c0d994129dff5d0d16b0b2bf2801a77df3a03b611fecf7aR48)) to avoid doing so at the service level. Do you think it is clear enough? If it doesn't, I can divide this into two different functions to have more control over the flow of messages.